apps/web/src/utils/keyboard.ts

```diff
@@ -18,8 +18,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 import EventManager from "@notesnook/core/utils/event-manager";
-
-const GlobalKeyboard = {};
+interface GlobalKeyboard {
+  addEventListener(
+    name: string,
+    handler: EventListenerOrEventListenerObject
+  ): void;
+  removeEventListener(
+    name: string,
+    handler: EventListenerOrEventListenerObject
+  ): void;
+}
+const GlobalKeyboard = {} as GlobalKeyboard;
```

Comment:
Define the `GlobalKeyboard` handlers in this object directly. That will simplify the types and also take care of the `as xyz` thing.